documentation/configuration/parameters/target.md

```diff
@@ -18,7 +28,8 @@ Special values:
 </ul>
 
 ## Default
-latest
+
+<code>latest</code> for migrations, <code>current</code> for undos.
```

Comment:
I think it would be clearer if we wrote `versioned migrations` and `undo migrations` since they're both a type of migration